packages/utils/src/initialize-cw-test-env/index.ts

```diff
@@ -0,0 +1,162 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// Keep dotenv import and config before everything else
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+import { patients } from "./patients";
+import axios from "axios";
+import * as AWS from "aws-sdk";
+import { getEnvVarOrFail } from "../shared/env";
+
+export type Doc = {
+  description: string;
+  fileName: string;
+};
+
+const AXIOS_TIMEOUT_MILLIS = 20_000;
+
+/**
+ * Initializes cw test env in staging
+ */
+
+const stagingTestAccountAPIKey = getEnvVarOrFail("COMMONWELL_ORG_PRIVATE_KEY");
+const facilityId = getEnvVarOrFail("FACILITY_ID");
+
+const orgId = getEnvVarOrFail("DOCUMENT_CONTRIBUTION_ORGANIZATION_ID");
+const orgName = getEnvVarOrFail("DOCUMENT_CONTRIBUTION_ORGANIZATION_NAME");
```

Comment:
From what I could tell, these could mismatch, no? I mean, ideally we would get one of those as a parameter and the rest we'd get from the DB, to avoid passing a `facilityId` that's not part of the `orgId`. For example, querying the facility and getting the org from there, and using the DB org's id and name.